.github/workflows/merge-validated-pr.yaml

```
@@ -6,6 +6,7 @@ on:
     branches: main
     paths:
       - 'model-output/**'
+      - '!**README**'
```

Since we're only including model-output we only need to / should exclude the `model-output/README.md` file.
```suggestion
      - '!model-output/README.md'
```